Source/WebKit/UIProcess/API/wpe/WebKitWebView.h

```diff
@@ -270,6 +270,7 @@ struct _WebKitWebViewClass {
     gboolean       (* show_option_menu)            (WebKitWebView               *web_view,
                                                     WebKitOptionMenu            *menu,
                                                     WebKitRectangle             *rectangle);
+    void           (* initialize_web_extensions)   (WebKitWebView               *web_view);
```

Comment:
Ditto, need to remove a padding byte. WPE has more padding than GTK, so here you'd remove `webkit_reserved5` (remove from the bottom).